helm-chart/kuberay-operator/README.md

```diff
@@ -18,13 +18,14 @@ version.BuildInfo{Version:"v3.6.2", GitCommit:"ee407bdf364942bcb8e8c665f82e15aa2
 
 To avoid duplicate CRD definitions in this repo, we reuse CRD config in `ray-operator`:
 ```console
-$ kubectl apply -k "../../ray-operator/config/crd"
+$ kubectl apply -k "github.com/ray-project/kuberay/ray-operator/config/crd?ref=v0.3.0"
 ```
 
 Please use command below:
 ```console
-$ helm install kuberay-operator . --values values.yaml --namespace ray-system --create-namespace
+$ helm install kuberay-operator --namespace ray-system --create-namespace https://github.com/ray-project/kuberay/releases/download/v0.3.0/helm-chart-kuberay-operator-0.1.0.tgz
```

Comment:
Something like this could be used to avoid hardcoding a release tag and chart version in the docs:
```suggestion
$ helm install kuberay-operator --namespace ray-system --create-namespace $(curl -s https://api.github.com/repos/ray-project/kuberay/releases/latest | grep '"browser_download_url":' | sort | grep -om1 'https.*helm-chart-kuberay-operator.*tgz')
```

Reply:
I think this makes more sense. Thanks!